### PR TITLE
Add basic menu for including our tools.

### PR DESCRIPTION
**Pull request by ShellyHerself**
Added a basic Krieg button that appears at the top of the window.

Contains:
 - (currently empty) 'Import' and 'Export' sub menus.
 - A 'Manual' button which links to the wiki for this git.
 - A 'GitHub' button which links to this repository.

Preview:
![img]

[img]: https://i.imgur.com/eLqF0vG.png

### CODE REVIEW

**Comment by ShellyHerself** [comment]
Closes #1 
Also updates/corrects the addon metadata.

File: menu/topbar_dropdown.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+    TOPBAR_MT_editor_menus.append(draw_krieg_button)
+
+def unregister():
+    from bpy.utils import unregister_class
+    from bpy.types import TOPBAR_MT_editor_menus
+
+    TOPBAR_MT_editor_menus.remove(draw_krieg_button)
+
+    for cls in reversed(classes):
```
Comment (Mimickal) [inline menu/topbar_dropdown.py]: Why do we reverse here? Can we add a short comment with that rationale?

File: menu/topbar_dropdown.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+def draw_krieg_button(self, context):
+    '''
+    When appended to another menu class this
+    function will execute after the built-in draw(),
+    and draw the 'Krieg' button.
+    '''
+    self.layout.menu(TOPBAR_MT_krieg.bl_idname, text=TOPBAR_MT_krieg.bl_label)
+
+def register():
+    from bpy.utils import register_class
```
Comment (Mimickal) [inline menu/topbar_dropdown.py]: Let's move these imports to the top of the file to avoid duplication.

File: __init__.py [hunk 3]
```diff
@@ -1,38 +1,30 @@
...
-    pass  # reload the addons to register/unregister
-else:
-    pass  # import the addons to register/unregister
-
-import bpy
-
-# Register
-classes = [
-]
+from .menu import topbar_dropdown
```
Comment (Mimickal) [inline __init__.py:13]: What does the dot do here?

File: menu/topbar_dropdown.py [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+
+# Enumerate all classes for easy register/unregister.
+classes = (
+    TOPBAR_MT_krieg,
+    TOPBAR_MT_krieg_import,
+    TOPBAR_MT_krieg_export,
+)
+
+# When appended to another menu class
```
Comment (Mimickal) [inline menu/topbar_dropdown.py]: This comment duplicates the info in the below docstring

**Review by Mimickal — CHANGES_REQUESTED** [review]

**Review by Mimickal — APPROVED** [review]

**Pull request merged**